e2e/cypress/integration/messaging/markdown_preview_inline_image_spec.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [number] indicates a test step (e.g. 1. Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Messaging', () => {
```

Comment:
maybe we could have a more descriptive title? something like `Markdown Image messaging`

Reply:
Thanks for the comments. This is on the same line as other UI Automation PRs. I am not sure the reason, but @saturninoabril asked me on other PRs to leave the description like that. I suppose here is the same.

Reply:
Yup, it should be 'Messaging'.  The main reason is for grouping of test, similar to how we group "tabs" in release testing spreadsheet.  That way, we can easily track which test belongs to what; initial step for reporting automation - marking passed/failed test from Cypress "daily/release test" into spreadsheet or test management in Jira.